ui_src/src/components/createStationForm/index.js

```diff
@@ -161,14 +161,14 @@ const CreateStationForm = ({ createStationFormRef, getStartedStateRef, finishUpd
         try {
             const data = await httpRequest('GET', ApiEndpoints.GET_AVAILABLE_REPLICAS);
             let replicas = [];
-            switch (data?.available_replicas) {
-                case 1:
+            switch (true) {
```

Comment:
switch(true)??

Reply:
@avrhamNeeman  This is to enable condition is switch case instead of changing to if statements

Reply:
@avrhamNeeman Fixed